packages/nova-react-test-utils/src/relay/test-utils.tsx

```diff
@@ -9,6 +9,8 @@ import { novaGraphql } from "./nova-relay-graphql";
 import { createMockEnvironment } from "relay-test-utils";
 import { RelayEnvironmentProvider } from "react-relay";
 import type { NovaMockEnvironment } from "./nova-mock-environment";
+import { NovaLocalizationProvider } from "@nova/react";
```

Comment:
Is this import used somewhere?

Reply:
No, I imported it, but the auto import wasn't removed.